eval/judge.py

```
@@ -0,0 +1,264 @@
+            continue
+
+        verdict_counts: dict[str, int] = {}
+        for v, _, _ in votes:
+            verdict_counts[v] = verdict_counts.get(v, 0) + 1
+        final_verdict = max(verdict_counts, key=lambda k: verdict_counts[k])
+        reasons = [r for _, r, _ in votes]
+
+        match_ids = [mid for _, _, mid in votes if mid]
+        matched_gt_id = match_ids[0] if match_ids else None
+
+        judge_type = "panel" if len(set(v for v, _, _ in votes)) == 1 else "panel-disputed"
```

Ruff (0.15.12)

[warning] Unnecessary generator (rewrite as a set comprehension)

Rewrite as a set comprehension

(C401)